### ISSUE TITLE
ember-jwt-decode fails to compile with Ember 3.0

### ISSUE THREAD

**Issue by BryanHunt** [Exelord/ember-jwt-decode#1]
I'm using ember-jwt-decode just fine in a project based on Ember 2.18, but when I started a new project with Ember 3.0, it fails to compile.  Here's the stack:

```
    at assertPath (path.js:28:11)
    at Object.join (path.js:1246:7)
    at Class.jwtDecodePath (/Users/bhunt/Nvidia/Code/nv-project/nv-project-ui/node_modules/ember-jwt-decode/index.js:23:17)
    at Class.treeForVendor (/Users/bhunt/Nvidia/Code/nv-project/nv-project-ui/node_modules/ember-jwt-decode/index.js:29:32)
    at Class._treeFor (/Users/bhunt/Nvidia/Code/nv-project/nv-project-ui/node_modules/ember-cli/lib/models/addon.js:556:33)
    at Class.treeFor (/Users/bhunt/Nvidia/Code/nv-project/nv-project-ui/node_modules/ember-cli/lib/models/addon.js:516:21)
    at project.addons.reduce (/Users/bhunt/Nvidia/Code/nv-project/nv-project-ui/node_modules/ember-cli/lib/broccoli/ember-app.js:613:25)
    at Array.reduce (<anonymous>)
    at EmberApp.addonTreesFor (/Users/bhunt/Nvidia/Code/nv-project/nv-project-ui/node_modules/ember-cli/lib/broccoli/ember-app.js:611:32)
    at EmberApp._processedVendorTree (/Users/bhunt/Nvidia/Code/nv-project/nv-project-ui/node_modules/ember-cli/lib/broccoli/ember-app.js:1098:24)
```

The problem appears to be that `this.app.project.nodeModulesPath` is undefined.

**Comment by slannigan** [comment]
Made a fix: https://github.com/Exelord/ember-jwt-decode/pull/2

**Comment by Exelord** [comment]
Fixed in `v0.1.1`. Thank you, guys!
